### PR TITLE
PPE8 fixes

### PR DESCRIPTION
**Pull request by royrapoport**

### CODE REVIEW

**Comment by royrapoport** [comment]
BTW, @jdq22 this is how I tested this:
1. Starting from an empty database downloaded channels, users, messages
2. Ran reports for user, channel, and slack

(Starting with the empty DB is how I found the bug that got fixed in the 'minor typo fix' commit above).  

It's not clear to me that a line-by-line review of this is a hugely positive way to spend your time :)

File: message.py [hunk 1]
```diff
@@ -23,17 +24,15 @@ def gsi(self, attname, wait=True):
         Create a GSI for attname, named AttnameIndex; wait until done if wait
         """
         idxname = attname.capitalize()
+        AttributeDefinitions = [{'AttributeName': attname, 'AttributeType': 'S'}]
```
Comment (jdq22) [inline message.py:27]: Is this naming style up to pep8?

File: utils.py [hunk 2]
```diff
@@ -51,9 +52,9 @@ def prune_empty(row):
     return new_row
 
 
-def chunks(l, n):
-    n = max(1, n)
-    return (l[i:i + n] for i in range(0, len(l), n))
+def chunks(mylist, n):
```
Comment (jdq22) [inline utils.py]: Would prefer this to be my_list to be consistent with the naming most other places

**Comment by jdq22** [comment]
> BTW, @jdq22 this is how I tested this:
> 
> 1. Starting from an empty database downloaded channels, users, messages
> 2. Ran reports for user, channel, and slack
> 
> (Starting with the empty DB is how I found the bug that got fixed in the 'minor typo fix' commit above).
> 
> It's not clear to me that a line-by-line review of this is a hugely positive way to spend your time :)

FWIW the line-by-line review took about 5 minutes. 😛

File: utils.py (hunk 2 above)
Comment (royrapoport) [reply utils.py]: Fixed in latest commit

File: message.py (hunk 1 above)
Comment (royrapoport) [reply message.py:27]: Maybe not @jdq22 -- I'm torn.  It's probably not the way Python wants it, but it is the name of the variables that the DDB boto3 library wants it.  Do we want to hew close to the library variable names or the PEP8 variable names?

Comment (royrapoport) [reply message.py:27]: Though actually I don't see conflict between these var names and PEP8 so I think we're good either way

**Review by jdq22 — APPROVED** [review]

**Pull request merged**